app/views/publish/courses/preview.html.erb

```diff
@@ -15,8 +15,6 @@
   <%= course.name_and_code %>
 </h1>
 
-<p class="govuk-body-l" data-qa="course__description"><%= course.description %></p>
```

Comment:
This seems like an unrelated removal?

Reply:
No, this is meant to be removed. It's not very clear in the ticket but on the design in Lucid this is removed.

On the ticket it says it's done when `The course summary is updated, with new information added with links to the new views, other information removed, and the order of the rows updated`

And the lucid board indicates that this should be removed https://lucid.app/lucidspark/971f26f4-cc5e-4993-8f70-efc85b3f0772/edit?invitationId=inv_f64ad81b-8b03-472c-898d-e6c398214ede&page=0_0#

![Screenshot from 2024-07-08 16-45-29](https://github.com/DFE-Digital/publish-teacher-training/assets/11318084/0df55904-9693-4ec2-81ba-d26c597d9c27)